pkgs/development/compilers/bluespec/default.nix

```diff
@@ -54,13 +61,13 @@ in stdenv.mkDerivation rec {
     verilog
   ];
 
-  patches = [
-    # drop stp support https://github.com/B-Lang-org/bsc/pull/31
-    (fetchpatch {
-      url = "https://github.com/flokli/bsc/commit/0bd48ecc2561541dc1368918863c0b2f4915006f.patch";
-      sha256 = "0bam9anld33zfi9d4gs502g94w49zhl5iqmbs2d1p5i19aqpy38l";
-    })
-  ];
+
+  postUnpack = ''
+    mkdir -p $sourceRoot/src/vendor/yices/v2.6/yices2
+    # XXX: only works because yices.src isn't a tarball.
```

Comment:
FWIW; is there a better way to do this?

Reply:
Does the source code need to be there, or can we patch the build system to accept an external yices, and set paths to header files and libraries (or have the buildsystem ask pkgconfig for it)

Reply:
Good point. I'll try that.